src/app.ts

```diff
@@ -157,6 +157,17 @@ async function downloadPage(title: string, link: string, doToggleMenu: boolean):
 
   await page.addStyleTag({ content: 'div[class^="styles__PrevNextButtonWidgetStyled"], div[class^="styles__Footer"], nav { display: none !important; }' });
 
+  await page.evaluate(() => {
+    var xPathResult = document.evaluate('//button[contains(@class, "AnimationPlus")]', document, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
```

Comment:
Changed to Dynamic depending on text AnimationPlus

Reply:
Worked nicely!
One suggestion, Please change the `var` to `const`. Same for line 163 (`let` to `const`).